app/pages/CreateUserPage.js

```diff
@@ -19,7 +19,10 @@ export class CreateUserPage extends BaseForm {
   }
 
   componentWillMount() {
-    const tokenString = this.props.match.params.tokenstring
+    const tokenString = this.props.location.pathname.substring(
```

Comment:
not sure - the router match on /createuser
should you call `loadRegistrationData()` if there is no token?
and would /createuser return a 404?